src/utils/logger/index.js

```diff
@@ -41,7 +41,7 @@ const initialState = String(
       localStorage.getItem(LS_KEY) : ''
 );
 
-const createLog = (namespace, options = {}) => new Logger(namespace, merge(options, defaultOptions));
+const createLog = (namespace, options = {}) => new Logger(namespace, merge(defaultOptions, options));
```

Comment:
Ain't this overriding the default options? We are assigning to a new variable in the `merge` method but objects are assigned per reference, not a copy. 

Are you sure of this? The fact that all errors were hidden and the show level is shown is enforced on purpose. We want a centralized knob to tweak ALL loggers.

Reply:
you are right. I will rollback this change, and recheck it in another moment.